manage/deploying/copy.rst

```diff
@@ -88,3 +88,8 @@ files using -a (archive) flag.
 .. code-block:: shell
 
    rsync -av --compress-level=9 var/blobstorage plone@server.com:/srv/plone/site/var
+   
+Upgrading a Plone Site
+===========================
+Install the new Plone version in another directory, and then copy in your Data.fs and blobstorage folders from the old site you're upgrading
```

Comment:
- Use double backticks for inline literals, files, and folders.
- Avoid contractions.
- Use punctuation.

```
Install the new Plone version in another directory, then copy your ``Data.fs`` and ``blobstorage`` folders from the old site you are upgrading.
```

Reply:
Resolved